dev/release/post-03-website.sh

```diff
@@ -55,14 +57,37 @@ rough_n_development_months=$((
 git_tag=apache-arrow-${version}
 git_range=apache-arrow-${previous_version}..${git_tag}
 
-committers_command_line="git shortlog -csn ${git_range}"
-contributors_command_line="git shortlog -sn ${git_range}"
-
-committers=$(${committers_command_line})
-contributors=$(${contributors_command_line})
+directories=("${ARROW_DIR}" "${ARROW_RS_DIR}")
```

Comment:
Should this have the ARROW_DF_DIR in it as well?

Reply:
Yep - I need to get in touch with the DF folks about their tagging conventions before I can do this.

Reply:
Ok cool, maybe a comment in here explaining that would be good to make it extra clear it's not an omission?

Reply:
Absolutely.

Reply:
I've added a comment and opened up a ticket here to clarify with the DataFusion folks: https://github.com/apache/arrow-datafusion/issues/770

Reply:
Per Nic, It seems like the `arrow-datafusion` releases will not necessarily be in sync with the `arrow` and `arrow-rs` releases, so we should leave DataFusion out of this for now and revisit later as noted in the `TODO` comments.